puppet/modules/slave/manifests/init.pp

```diff
@@ -229,10 +233,9 @@
   include slave::java
 
   # Databases
-  include slave::mysql, slave::postgresql
-  slave::db_config { 'mysql': }
+  include slave::mysql
+  include slave::postgresql
   slave::db_config { 'sqlite3': }
```

Comment:
I think this can be `ensure: absent` these days

Reply:
SQLite is still used in the 2.0 jobs. https://github.com/theforeman/foreman-infra/pull/1497 would probably solve it, but that PR is slow to be merged.

Reply:
it's merged now.

Reply:
Added as a separate commit now.